aws-elasticbeanstalk-agent/src/main/java/jetbrains/buildServer/runner/elasticbeanstalk/LoggingDeploymentListener.java

```diff
@@ -54,31 +54,30 @@
   void createVersionStarted(@NotNull String applicationName, @NotNull String versionLabel,
                             @NotNull String s3BucketName, @NotNull String s3ObjectKey) {
     open(CREATE_VERSION);
-    log(String.format("Creating application %s version %s with bucket %s and key %s", applicationName, versionLabel, s3BucketName, s3ObjectKey));
+    log(String.format("Creating application %s version %s with bucket %s and key %s.", applicationName, versionLabel, s3BucketName, s3ObjectKey));
   }
 
   @Override
   void createVersionFinished(@NotNull String applicationName, @NotNull String versionLabel,
                              @NotNull String s3BucketName, @NotNull String s3ObjectKey) {
-    log(String.format("Created application %s version %s with bucket %s and key %s", applicationName, versionLabel, s3BucketName, s3ObjectKey));
+    log(String.format("Created application %s version %s with bucket %s and key %s.", applicationName, versionLabel, s3BucketName, s3ObjectKey));
     close(CREATE_VERSION);
   }
 
   @Override
   void deploymentStarted(@NotNull String applicationName, @NotNull String environmentName, @NotNull String versionLabel) {
     open(UPDATE_ENVIRONMENT);
-    log(String.format("Started deployment of application %s version %s to %s", applicationName, versionLabel, environmentName));
+    log(String.format("Started deployment of application %s version %s to %s.", applicationName, versionLabel, environmentName));
   }
 
   @Override
   void deploymentWaitStarted(@NotNull String environmentName) {
-    log("Waiting for deployment finish");
-    log(String.format("Waiting for deployment on environment %s", environmentName));
```

Comment:
This seemed duplicate with the method directly below this one. It caused a bunch of spam in the build log.